src/opm/parser/eclipse/EclipseState/Schedule/Group/GuideRate.cpp

```diff
@@ -22,74 +22,158 @@
 
 namespace Opm {
 
+double GuideRate::GuideRateValue::eval(GuideRateModel::Target target_arg) const
```

Comment:
The `target_arg` parameter is [unused](https://ci.opm-project.org/job/opm-common-PR-builder/lastBuild/gcc/source.48458f53-bdc6-471e-8bf9-735dbe1de789/#25) in this function.  Is that intentional?

Reply:
Thanks - that was a bug